### DISCUSSION TITLE
Save A Navigation Route For Later Use

### DISCUSSION THREAD

**Discussion by Sonwon1** [osmandapp/OsmAnd#17819] [Q&A] [answered]
I am planning a 14-21 day motorcycle trip. I wanted to use OsmAnd to save the navigation route, one for each day of the trip. I tried creating a navigation route however I do not see any way to save it for later usage. I also tried Plan A Route and do not see anyway to search for the next POI I am driving too.

Is this possible with OsmAnd, if so how?

**Comment by mikehgentry** [comment, marked as answer]
For navigation just go to 'Details' then hit the download button in the top right, it'll save it as a track.

For 'Plan a route' mode it should just save it when you hit done.

Easiest way to find POIs is to hit search -> categories -> custom search, add every type of POI you want, hit show, then save with the buttons in the top right. You can then use that custom filter like a normal category, and make it visible in either navigation or plan a route mode.

**Reply by Sonwon1** [reply]
Thank you very much. That works! It is even better than importing a GPX file made with Garmin BaseCamp. Bonus, I do not need to use 'Attach To The Roads'. I did try 'Attach To The Roads' the first time and OsmAnd stuck at less than 5% for over 4 minutes when I decided I should try it without, it worked perfectly. The route for Day 1 is 384 km, 3 hours and 24 minutes.

This is not intuitive though, the interface design needs some work. And I think 'Attach To The Roads' could possibly be broken for big GPX files?

**Reply by mikehgentry** [reply]
Being a cyclist I don't ride ~400km in a day, so wouldn't have noticed! :)

It's often worth using attach to the roads - it gives you additional information like a breakdown of surface type, and significantly improves the turn by turn instructions. You may have to chop the GPX up into pieces to get it to work (I could imagine it running out of memory?). Still, if there's a GPX that it consistently fails on, probably worth attaching it to a bug report.

And yeah, the interface is improving, but there are still some things that you must essentially learn by rote, because they're a pain to find.

**Reply by Sonwon1** [reply]
I just checked the details for the road surface, smoothness, steepness and turn by turn are all good. I am quite pleased with OsmAnd at the moment. I was hoping it could replace a Garmin Zumo XT and I have now gotten pass the show stoppers.

**Reply by mikehgentry** [reply]
Yeah, attach to the roads is only relevant when the route has been built by something other than OsmAnd's inbuilt router (so importing a GPX, or using BRouter's integration into OsmAnd etc.)

Glad you got it working anyway, and happy travels :)

**Reply by Sonwon1** [reply]
Thank you, happy travels to you too!
